src/k8s/components/jaeger/documentation.md

```diff
@@ -0,0 +1,132 @@
+# Table of Contents
+
+- [Table of Contents](#table-of-contents)
+- [Deployment](#deployment)
+  - [Containers](#containers)
+    - [`jaeger`](#jaeger)
+      - [image](#image)
+      - [additional environment variables](#additional-environment-variables)
+      - [resources](#resources)
+      - [additional volume mounts](#additional-volume-mounts)
+  - [Additional SideCar Containers](#additional-sidecar-containers)
+
+# Deployment
+
+## Containers
+
+### `jaeger`
+
+#### image
+
+**Customization snipppet**:
+
+```dhall
+with jaeger.Deployment.Containers.jaeger.image = <Image>
+```
+
+**Default value**:
+
+```dhall
+Image::{
+      , name = "sourcegraph/jaeger"
+      , registry = Some "index.docker.io"
+      , digest = Some
+          "abc123DEADBEEF"
+      , tag = "3.20.1"
+      }
+```
+
+_The default values of `digest` and `tag` will vary depending on the release in question (e.x. `tag` will be `"3.21.1"` for the `3.21` Sourcegraph release, etc.)._
+
+<!-- TODO: Should we even be documenting this, or should we just direct people to the global options? -->
+
+#### additional environment variables
+
+**Customization snipppet**:
+
+```
+
+with jaeger.Deployment.Containers.jaeger.additionalEnvVars = [ Sourcegraph.Util.EnvToK8s { name = "fooKey", value = "fooValue" } ]
```

Comment:
FYI - I rewrote this documentation to just refer to the variable with the whole package in it (which includes this utility function, the generate function, the configuration record, and the whole kubernetes schema)

Reply:
nice